synapse/lib/cell.py

```diff
@@ -1204,11 +1211,11 @@ async def __anit__(self, dirn, conf=None, readonly=False, parent=None):
         await self._initCellBoot()
 
         # we need to know this pretty early...
-        self.ahasvcname = None
         ahaname = self.conf.get('aha:name')
         ahanetw = self.conf.get('aha:network')
         if ahaname is not None and ahanetw is not None:
             self.ahasvcname = f'{ahaname}.{ahanetw}'
+            s_logging.setLogInfo('service', self.ahasvcname)
```

Comment:
Should we be setting this in `initFromArgV` to avoid 1) potential nested cells from overwriting the parent `service` setting if they were configured with these values & 2) avoid global state mutation in testing?

Reply:
Def open to discussion. I think some global state mutation is going to be difficult to avoid, i just wanted to make sure this is ALWAYS set ( even for tests which don't initialize the service via `initFromArgv`.